swap_meet/item.py

```diff
@@ -1,2 +1,32 @@
+import uuid
+import math
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):           
+        self.id = id if id else uuid.uuid4().int
+        if type(self.id) != int:
+            raise TypeError("Id should be interger")
+        self.condition = condition
+        
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}"
+    
+    def get_category(self):
+        return self.__class__.__name__
+
+    def condition_description(self) : 
+        # Make sure condition is a integer between zero and five
+        condition = max(0, min(5, math.floor(self.condition)))
+        description_dict = {
```

Comment:
great use of a `dict` here. And `if/else` block would take O(n) worse case to fall through to the last condition!

Plus, nice formatting of the contents of the `dict` here